### DISCUSSION TITLE
iOS build for simulator throwing error 'box2d/box2d.h' file not found

### DISCUSSION THREAD

**Discussion by meehjeevan** [axmolengine/axmol#1951] [Q&A]
I used this command for build
axmol build -p ios -a x64 -c
I tried with this one too 
axmol build -p ios -a x64 -sdk simulator -c

but in both cases I'm getting this error while trying to run the project on simulator.
![Screen Shot 2024-05-29 at 3 25 52 PM](https://github.com/axmolengine/axmol/assets/6400139/994f1b6f-6bba-4a21-8a58-60c41e93b452)
And sometimes these errors too:
![Screen Shot 2024-05-29 at 5 17 37 PM](https://github.com/axmolengine/axmol/assets/6400139/1b5ca136-a323-4e7b-9071-036d4443e402)
![Screen Shot 2024-05-29 at 5 17 00 PM](https://github.com/axmolengine/axmol/assets/6400139/f1e832c5-3f83-40dc-8604-27be59e23588)

How do I solve it ? 

OS version: 10.15.7
Xcode: 12.4
cmake: 3.29.3

**Comment by halx99** [comment]
That's strange, because our ci works fine.
